CONTRIBUTING.md

```diff
@@ -69,4 +75,4 @@ Don't:
 
 ## Merging Pull Requests (PRs)
 
-The preferred order of [merging PRs](https://help.github.com/articles/about-pull-request-merges/) is: prefer rebase, else squash if you'd like to clean up the history before merging. Avoid doing a merge commit.
\ No newline at end of file
+The preferred order of [merging PRs](https://help.github.com/articles/about-pull-request-merges/) is: prefer rebase, else squash if you'd like to clean up the history before merging. Avoid doing a merge commit.
```

Comment:
Is it just _my_ eyes? What has changed? I can't see the difference in this pair of lines... ? 😖